src/ralph/virtual/management/commands/openstack_sync.py

```diff
@@ -361,6 +368,75 @@ def _get_hypervisor(host_name, server_id):
             ))
             return None
 
+    def _match_physical_and_cloud_hosts(self):
+        """Connect CloudHosts and DC assets according to data from Ironic."""
+
+        provider = getattr(
```

Comment:
if you'd use this flag (`--match-ironic-physical-hosts` or similar), this wouldn't be necessary - just use current provider passed in args